plugin-template

```diff
@@ -90,6 +90,11 @@ def main():
                         help=textwrap.dedent('''\
                             Generate or update travis configuration files.
 
+                        '''))
+    parser.add_argument('--github', action='store_true',
```

Comment:
maybe `--ci`? (Not a strong opinion)

Reply:
I initially fixed this and then changed my mind, let's decide this once we have a concrete plan about supporting other CI services.